e2e/test/scripts/create_cluster.sh

```diff
@@ -7,7 +7,8 @@ set -x
 
 export LINODE_API_TOKEN="$1"
 export CLUSTER_NAME="$2"
-export K8S_VERSION="$3"
+export REGION="$3"
+export K8S_VERSION="$4"
```

Comment:
In order to not break any possible existing automation I think we should try to keep the params in the same order. So we can try this instead.

```suggestion
export K8S_VERSION="$3"
export REGION="$4"
```